### PR TITLE
WIP Dark mode enhancements

### PR DESCRIPTION
**Pull request by robbieaverill**
Since login-forms announces support for dark mode, we should have at least minimum support for it in these modules too.

This is a WIP pull request, there's some code here that belongs in TOTP but I didn't have WiFi at the time to build it there.

See #139

### CODE REVIEW

File: client/src/styles/mfa.scss [hunk 1]
```diff
@@ -51,5 +51,19 @@ body.mfa {
...
-  color: $color-grey-text;
+  color: var(--color-text);
+
+  // Dark mode only styles (for things like Bootstrap that don't have their own components already)
+  @media (prefers-color-scheme: dark) {
+    .btn-secondary {
+      color: var(--color-text);
+    }
+
+    .mfa-totp__scan-left {
```
Comment (robbieaverill) [inline client/src/styles/mfa.scss]: Needs to be moved to TOTP

File: client/src/styles/mfa-variables.scss [hunk 2]
```diff
@@ -44,20 +49,56 @@ $loading-indicator-animation: spinner-loader $loading-indicator-speed infinite l
 
 // Authentication method tiles
 $tile-border-color: $color-grey-border-light;
+$tile-border-color-dark: $color-grey-border-light;
 $tile-border-color-hover: lighten($tile-border-color, 9.5);
+$tile-border-color-hover-dark: darken($tile-border-color-dark, 40);
 $tile-border-color-active: rgb(0, 111, 199);
+$tile-border-color-active-dark: var(--color-green);
```
Comment (Cheddam) [inline client/src/styles/mfa-variables.scss]: This should probably be a lighter shade of blue, rather than a totally different colour?

**Comment by Cheddam** [comment]
Just had another thought about this - I haven't tested what it does in the CMS right, but I think we'll need to explicitly disable the Dark Mode styles there.

**Comment by robbieaverill** [comment]
I think @Cheddam's work is superseding this

**Pull request closed without merging**